WordPress/Classes/Utility/CoreData/ContextManager.swift

```diff
@@ -222,7 +222,7 @@ private extension ContextManager {
 
 private extension ContextManager {
     static func createPersistentContainer(storeURL: URL, modelName: String) -> NSPersistentContainer {
-        guard var modelFileURL = Bundle.main.url(forResource: "WordPress", withExtension: "momd") else {
+        guard var modelFileURL = Bundle(for: ContextManager.self).url(forResource: "WordPress", withExtension: "momd") else {
```

Comment:
Just noting this change. I tested it and it works as `Bundle(for: ContextManager.self)` returns `Bundle.main`. I'm not sure if this change belongs in this PR, but I don't have strong feelings about. If it gets merged as part of it, it's totally fine.